src/Sets/HPolyhedron.jl

```diff
@@ -483,7 +483,7 @@ Otherwise, we set up the LP internally.
 function isempty(P::HPoly{N},
                  witness::Bool=false;
                  use_polyhedra_interface::Bool=false,
-                 solver=default_lp_solver(N),
+                 solver=nothing,
```

Comment:
I guess the docs need to be adapted now.

Reply:
done. on the other hand, having `(optional, default: `nothing`)` is sort of meaningless so i added a comment in that line